classes/core/PKPHtmlSanitizer.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+/**
+ * @file classes/core/PKPHtmlSanitizer.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class PKPHtmlSanitizer
+ *
+ * @ingroup core
+ *
+ * @brief Wrapper on the top of Symfony's HtmlSanitizer implementation
+ *
+ */
+
+namespace PKP\core;
+
+use Illuminate\Support\Str;
+use Illuminate\Support\Collection;
+use PKP\config\Config;
+use Symfony\Component\HtmlSanitizer\HtmlSanitizer;
+use Symfony\Component\HtmlSanitizer\HtmlSanitizerConfig;
+use Symfony\Component\HtmlSanitizer\Reference\W3CReference;
+
+class PKPHtmlSanitizer
+{
+    /**
+     * Collection of valid element specify by W3C Sanitizer API
+     * 
+     * @see const HEAD_ELEMENTS and BODY_ELEMENTS at \Symfony\Component\HtmlSanitizer\Reference\W3CReference
+     * @see https://wicg.github.io/sanitizer-api/#default-configuration
+     */
+    protected Collection $w3cValidElements;
+
+    /**
+     * Instance of HtmlSanitizerConfig config
+     */
+    protected HtmlSanitizerConfig $htmlSanitizerConfig;
+
+    /**
+     * Instance of HtmlSanitizer
+     */
+    protected HtmlSanitizer $htmlSanitizer;
+
+    /**
+     * Create a new instance
+     * 
+     * @param string $allowable Config key such security.[allowed_html/allowed_title_html]
```

Comment:
It's confusing to have this operate in two modes. I think you could just move the `Config` interaction into `PKPString`, and make this function always accept a string of allowable elements. (Also I think `attribited` should be `attributes`.)

Reply:
done .